ballerina-tests/tests/05_invalid_resource_service.bal

```diff
@@ -46,12 +46,6 @@ service "HelloWorld98" on ep5 {
         } else {
             displayAge = 1;
         }
-        grpc:Error? err = caller->sendInt(displayAge);
-        if err is grpc:Error {
-            log:printError(err.message(), 'error = err);
-        } else {
-            log:printInfo("display age : " + displayAge.toString());
-        }
-        checkpanic caller->complete();
+        checkpanic caller->sendInt(displayAge);
```

Comment:
```suggestion
        checkpanic caller->sendInt(displayAge);
        checkpanic caller->complete();
```

Reply:
Should we need to complete all the time since it automatically calls complete when we don't specify?.